solutions/pandas_tests/Makefile

```diff
@@ -0,0 +1,76 @@
+# Some tests in files listed in selective_tests fail/ error out on execution.
+# These failures and error are caused by one of below:
+# 1. pypaper linux specific errors, documented
+# here - https://pyperclip.readthedocs.io/en/latest/#not-implemented-error
+# 2. s3fs error (which is currently an optional file handling mechanism in pandas)
+# https://bleepcoder.com/evalml/557744402/unit-test-fails-after-upgrading-to-pandas-1-0-0
```

Comment:
According to the info/URL in this link, the issue was fixed on Panda's Main branch around Jun/July, 2020. Is the pip installed pandas release for Python3.9 and old version without the fix?

Reply:
When I install python3 and pandas on the ubuntu VM (without docker images) and docker image ubuntu:18.04 (with apt-get update && apt-get install -y python3, pip3 etc), the s3 errors don't occur. It's only in this docker image that this issue occurs. I am guessing python:3.9-slim needs to be updated..

I decided against using ubuntu:18.04 because it was taking over 10 mins to do docker build, increasing the testing time.